### ISSUE TITLE
ITS cutadapt file empty

### ISSUE THREAD

**Issue by kbeals2** [benjjneb/dada2#1209]
Hi there, 

I am running into a bug with my ITS pipeline in which the following cutadapt loop is only returning two .txt files each named for the primers. It's not creating any files files for the cutadapted sequences. I don't have any error messages in the cutadapt output. I'm using the most recent version of cutadapt (3.0), my files fastq files are all compressed, and I have no spaces or weird characters in my directory or file names. I've tried this pipeline on ITS sequences for other completed projects to test if this is an error with my current sequences, and I run into the same issue. Any thoughts??

for(i in seq_along(FWD_reads)) {
  system2(cutadapt, args = c(R1_flags, R2_flags, "-n", 2, "-o", FWD_cut[i], "-p", REV_cut[i], FWD_filt[i], REV_filt[i])) 
}

**Comment by benjjneb** [comment]
Have you tested it on uncompressed input files? Occasionally people have run into issues with cutadapt not recognizing the compression.

**Comment by kbeals2** [comment]
Thanks for the suggestion. Unfortunately using uncompressed files didn't work.

**Comment by benjjneb** [comment]
What happens when you try to run cutadapt on a single file, preferably in the shell?

**Comment by kbeals2** [comment]
I ran this separately for each sample in the shell and it worked: 

cutadapt samplename_R1.fastq.gz samplename_R2.fastq.gz -o samplename_R1_cut.fastq.gz -p samplename_R2_cut.fastq.gz -g ^AGCCTCCGCTTATTGATATGCTTAART -a AGWGATCCRTTGYYRAAAGTT$ -G ^AACTTTYRRCAAYGGATCWCT -A AYTTAAGCATATCAATAAGCGGAGGCT$

The input files were in my "filtered" foldered, so I manually moved the output cut files into the previously made "cutadapt" folder. I then re-assigned FWD_cut and REV_cut to these new cutadapted files. I checked that the primers were succesfully removed and I was able to get back to the DADA2 pipeline :)

 However, in the future I would prefer to use the cutadapt loop in R. I'm still not sure why that is not running properly.
